kcbq-connector/pom.xml

```diff
@@ -67,6 +67,10 @@
             <groupId>io.debezium</groupId>
             <artifactId>debezium-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
```

Comment:
Do we need a cache? I was checking the SchemaManager, and I don't have the impression we are going to have such an amount of BigQuery metadata instances to cause a memory issue?

If that's the case, I'd rather use just a `HashMap` instead of a library.

Reply:
Hi @robertotena,

@C0urante suggested that we cache the tables to prevent a large number of requests to BigQuery.

The reason I opted for a library is to allow configuration for the size of the cache and a cache expiry per entry. Currently I have set it so that you can configure against the number of hours that you would like an entry to be held within the cache. We can expand on this configuration to allow users to choose against the enumeration values which are present in java.util.concurrent.TimeUnit.

Reply:
I don't see an advantage to expiring entries from the cache, considering that the partitioning type for a table can't be modified after it's created, and it would take a gigantic number of tables in the cache to incur a non-negligible performance hit. I think a `HashMap` is fine here too 👍